src/main/java/uk/gov/cslearning/record/domain/Invite.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.cslearning.record.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Invite {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private int eventId;
```

Comment:
Can you change this to be a relationship that returns an Event rather than simply return an eventId?